code/game/objects/items/weapons/tools.dm

```diff
@@ -641,8 +639,9 @@
 	icon = 'icons/obj/tools.dmi'
 	icon_state = "powerdrillyellow"
 	var/drillcolor = null
-	force = 3
-	w_class = 2
+	force = 10
```

Comment:
I think this should be on the same level as the others, so like 8 force. It's not exactly physically different or stronger.